src/components/Nav.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './Nav.css';
+
+export const Nav = () => {
```

Comment:
it would be nice to discuss if you guys are using functions or arrow functions with the rest of your team